app/models/vm.rb

```diff
@@ -121,6 +121,12 @@ def supported_consoles
     }
   end
 
+  def v2v_migrated?
+    ServiceResource.where(:resource => self, :status => 'complete').any? do |resource|
+      ServiceTemplate.find(resource.service_template_id).kind_of?(ServiceTemplateTransformationPlan)
```

Comment:
why another query? Can be `resource.service_template.kind_of?(ServiceTemplateTransformationPlan)`.